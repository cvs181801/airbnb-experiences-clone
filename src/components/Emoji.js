import React from 'react'

export default function Emoji(props) {
    const anEmoji = props.isFilled ? 'C' : 'Not an emoji';
    return (
        <div>
            <h1
                onClick={props.handleClick2}
            >{anEmoji}</h1>
        </div>
    )
}
