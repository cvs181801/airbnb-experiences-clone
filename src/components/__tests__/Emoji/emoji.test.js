import {render, screen, cleanup} from '@testing-library/react'
import renderer from 'react-test-renderer'
import '@testing-library/jest-dom'
import Emoji from '../../Emoji'
import * as React from 'react';

afterEach(()=> {
    cleanup();
})

test("should render heart emoji", ()=>{
    render(<Emoji/>)
    const heart = screen.getByTestId("anEmoji")
    expect(heart).toBeInTheDocument();

    
});

test('render matches snapshot', ()=>{
    const component = renderer.create(<Emoji/>)
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});