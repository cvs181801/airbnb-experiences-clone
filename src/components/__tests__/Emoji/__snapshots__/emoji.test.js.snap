// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render matches snapshot 1`] = `
<div>
  <h1
    data-testid="anEmoji"
  >
    Not an emoji
  </h1>
</div>
`;
